Ports/opentyrian-data/package.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=opentyrian-data
+version=1.0.0
+workdir=.
+files="http://camanis.net/tyrian/tyrian21.zip tyrian21.zip 7790d09a2a3addcd33c66ef063d5900eb81cc9c342f4807eb8356364dd1d9277"
+auth_type=sha256
+
+build() {
+    run_nocd unzip -o tyrian21.zip && cd tyrian21 && rm -v *.exe
```

Comment:
```suggestion
    run_nocd unzip -o tyrian21.zip
    run_nocd rm -v tyrian21/*.exe
```

this should work, right?